gapis/service/path/path.proto

```diff
@@ -152,6 +152,7 @@ message Events {
     bool push_user_markers = 8;
     bool pop_user_markers = 9;
     bool framebuffer_observations = 10;
+    bool all_commands = 11;
```

Comment:
This purpose of this is so that anything (code / interface etc) that relies on events, does not have to split their implementation into 2. 1 to handle events, and 1 to handle "everything".

Eg: Video.go:83
```go
	if verb.Commands {
		requestEvents.LastInFrame = false
		requestEvents.AllCommands = true
	}
```

Is the only change on the client that is necessary to handle either of these cases, rather than having to create a secondary loop that works on Commands instead of returned Events.